cvpy/tests/test_cvat_project.py

```diff
@@ -214,14 +214,342 @@ def test_cvat_project_resume(self):
+        self.assertEqual(second_image['_nObjects_'], 1)
+        self.assertAlmostEqual(second_image['_Object0_xMin'], 321.936526, 2)
+        self.assertAlmostEqual(second_image['_Object0_yMin'], 140.750162, 2)
+        self.assertAlmostEqual(second_image['_Object0_xMax'], 446.329383, 2)
+        self.assertAlmostEqual(second_image['_Object0_yMax'], 334.250162, 2)
+        self.assertTrue(np.isnan(second_image['_Object1_xMin']))
+
+        # Assert that the third image's annotations are correct.
+        third_image = output_annotations.to_frame().iloc[2]
+        self.assertEqual(third_image['_Object0_'], 'Mountain')
+        self.assertEqual(third_image['_nObjects_'], 1)
+        self.assertAlmostEqual(third_image['_Object0_xMin'], 385.496591, 2)
+        self.assertAlmostEqual(third_image['_Object0_yMin'], 240.609091, 2)
+        self.assertAlmostEqual(third_image['_Object0_xMax'], 702.985227, 2)
+        self.assertAlmostEqual(third_image['_Object0_yMax'], 306.086363, 2)
+        self.assertTrue(np.isnan(third_image['_Object1_xMin']))
+
+        # Assert that the fourth image's annotations are correct.
+        fourth_image = output_annotations.to_frame().iloc[3]
+        self.assertEqual(fourth_image['_Object0_'], 'Person')
+        self.assertEqual(fourth_image['_Object1_'], 'Person')
+        self.assertEqual(fourth_image['_nObjects_'], 2)
+        self.assertAlmostEqual(fourth_image['_Object0_xMin'], 439.770779, 2)
+        self.assertAlmostEqual(fourth_image['_Object0_yMin'], 453.183117, 2)
+        self.assertAlmostEqual(fourth_image['_Object0_xMax'], 558.991558, 2)
+        self.assertAlmostEqual(fourth_image['_Object0_yMax'], 599.228571, 2)
+        self.assertAlmostEqual(fourth_image['_Object1_xMin'], 1846.575974, 2)
+        self.assertAlmostEqual(fourth_image['_Object1_yMin'], 733.351948, 2)
+        self.assertAlmostEqual(fourth_image['_Object1_xMax'], 2016.465584, 2)
+        self.assertAlmostEqual(fourth_image['_Object1_yMax'], 933.046753, 2)
+
+        ## Assert that the fifth image's annotations are correct.
+        fifth_image = output_annotations.to_frame().iloc[4]
+        self.assertEqual(fifth_image['_Object0_'], 'Mountain')
+        self.assertEqual(fifth_image['_nObjects_'], 1)
+        self.assertAlmostEqual(fifth_image['_Object0_xMin'], 981.298828, 2)
+        self.assertAlmostEqual(fifth_image['_Object0_yMin'], 893.506836, 2)
+        self.assertAlmostEqual(fifth_image['_Object0_xMax'], 1656.623047, 2)
+        self.assertAlmostEqual(fifth_image['_Object0_yMax'], 1238.960938, 2)
+        self.assertTrue(np.isnan(fifth_image['_Object1_xMin']))
 
 if __name__ == '__main__':
     if len(sys.argv) > 1:
         TestCVATProject.cas_host = sys.argv.pop(1)
         TestCVATProject.cas_port = sys.argv.pop(1)
         TestCVATProject.cas_username = sys.argv.pop(1)
         TestCVATProject.cas_password = sys.argv.pop(1)
-        TestCVATProject.cas_protocol = sys.argv.pop(1)
```

Comment:
Can you keep this line: protocol=TestCVATProject.cas_protocol?

When running the test, you will need to pass in the protocol: either cas or http.